anyblock-gasprice/src/adapter.ts

```diff
@@ -0,0 +1,13 @@
+import { Requester } from '@chainlink/external-adapter'
+import { Config, ExecuteWithConfig, ExecuteFactory } from '@chainlink/types'
+import { makeConfig } from './config'
+import { gasprice } from './endpoint'
+
+export const execute: ExecuteWithConfig<Config> = async (request, config) => {
+  Requester.logConfig(config)
+  return await gasprice.execute(request, config)
```

Comment:
Maybe merge with the `anyblock-uniswap-vwap` endpoint as well, as you did with `bravenewcoin`?

Reply:
Unfortunately can't yet because `anyblock-gasprice` uses `endpoint` as a param :disappointed: